utils/sedutil/Makefile

```
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sedutil
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/Drive-Trust-Alliance/sedutil
+PKG_MIRROR_HASH:=e11333bfa0760a46cbebcba35360e0f076e6219eb38ce1545179b8741476668a
+PKG_SOURCE_DATE:=2022-12-27
+PKG_SOURCE_VERSION:=7a0cda7f60cce346f72466e61ce006e5ea48fbc0
+
+PKG_LICENSE_FILES:=README.md
+PKG_LICENSE:=GPL-3.0-or-later
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
```

```suggestion
PKG_BUILD_PARALLEL:=1
PKG_BUILD_FLAGS:=lto
```

use new `PKG_BUILD_FLAGS` for lto flag